include/tuvx/cross_section_params.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once 
+
+#include <string>
+
+#include <vector>
+
+namespace tuvx {
+  public:
+     
+}
```

Comment:
Is this working in progress? Would it make sense that you submit this when it's complete?